UI/FrontEnd.py

```diff
@@ -0,0 +1,82 @@
+from tkinter import Frame, Label, Button, OptionMenu, Tk, StringVar, RAISED
+from tkinter.colorchooser import askcolor
+from VolumeLogic import ChangedValue, VolumeMixer
+from SystemConnectors import WindowsConnection
+from functools import partial
+
+
+class FrontEnd():
+    """Handles the Settings UI"""
+    
+    def __init__(self, volume_mixer):
+        """
+        Initializes FrontEnd
+        :param volume_mixer: Volume Mixer used to communicate with rest of application
+        """
```

Comment:
New line between comment and param. Only going to write it once, but applies to multiple places.